xapian-applications/omega/omindexcheck.cc

```diff
@@ -170,8 +181,14 @@ main(int argc, char** argv)
 	Xapian::Document doc = db.get_document(did);
 	auto iter = tests.find(url);
 	if (iter != tests.end()) {
-	    succeed &= compare_test(iter->second, doc, url);
+	    if (startswith(iter->first, "audio") ||
+		startswith(iter->first, "video"))
+		skip &= compare_test(iter->second, doc, url);
+	    else
+		succeed &= compare_test(iter->second, doc, url);
```

Comment:
I think we need to find a better approach than this.

As written, the test can never fail for an audio or video file, but there are test results which are clearly a failure because we should either get all the terms that come from a plugin or none of them.  So for example, for `audio/file_example_OOG_1MG.ogg` if we get `Akevin` we should also get `Simpact`.

You need to think carefully about what cases might happen due to a missing plugin and what can't, so that the test can actually catch problems.

Also, the variable `skip` seems to be confusingly named - it looks to me like `skip` being `false` means we're skipping the tests if they'd otherwise pass.  The sense seems reversed from what I'd expect.

(And a coding style point - it's clearer to put `{`...`}` on the blocks for an `if` where the condition has been split over multiple lines like this.)